python/src/gen3authz/utils.py

```diff
@@ -0,0 +1,36 @@
+from functools import wraps
+
+
+class Return(Exception):
+    __slots__ = ("value",)
+
+    def __init__(self, value):
+        self.value = value
+
+
+def return_(value=None):
```

Comment:
why is this necessary?

Reply:
it's a little confusing to replace all the returns with a function, it might be cleaner to use a decorator or something for this?

Reply:
Oh this is a Python 2 compatibility fix - `return` is not allowed in generator functions. But as we require Python 3.6 (is this true?), I think this is not necessary.